package.json

```diff
@@ -76,7 +76,7 @@
     "uuid": "^9.0.1"
   },
   "peerDependencies": {
-    "@stream-io/openai-realtime-api": "prerelease"
+    "@stream-io/openai-realtime-api": "^0.1.0"
```

Comment:
If we use a pinned version in dev dependency shouldn't we use a pinned version here?

Reply:
I wanted it to be a bit relaxed, so we don't have to release new node-sdk versions if we fix a bug in the other package.
I pinned in dev dependencies, although we can have ^0.1.0, and `yarn.lock` will pin it instead.

Reply:
Yep, I thought you pinned because it's `0.x.x` release so every release could be breaking, that's what confused me

Reply:
I think we should be nice and bump the minor for all possible breaking changes while in `0.x.x` range.
e.g - 0.1.1 (non breaking), 0.2.0 (possibly breaking)

Reply:
`^0.1.0` still includes breaking ranges, but I'm fine with the change, I just thought using `0.1.0` in one place but `^0.1.0` in the other might be a mistake, so just wanted to ask not to miss something 🙂

Reply:
In `0.x.x` ranges, `^0.1.0` is identical to `~0.1.0`. That's what semver says :D
I'll make it more explicit though.

Reply:
oh, I didn't know that

Reply:
Changed here: https://github.com/GetStream/stream-node/pull/92/commits/d7b08ed4bf519fadacffe9f87408d2d9fc444ab9

Reply:
Thank you